lbrynet/core/Wallet.py

```diff
@@ -975,6 +976,11 @@ def _update_balance(self):
             lambda result: Decimal(result['confirmed']) + Decimal(result.get('unconfirmed', 0.0)))
         return d
 
+    @defer.inlineCallbacks
+    def get_max_usable_balance_for_claim(self, claim_name):
+        amt = yield self._run_cmd_as_defer_to_thread('get_max_spendable_amt_for_claim', claim_name)
+        defer.returnValue(amt)
```

Comment:
You can simplify this to just:
```python
def get_max_usable_balance_for_claim(self, claim_name):
    return self._run_cmd_as_defer_to_thread('get_max_spendable_amt_for_claim', claim_name)
```

`@defer.inlineCallbacks` turns a method into something that returns a `Deferred` but `_run_cmd_as_defer_to_thread()` already returns a `Deferred`, so you can skip all the indirection and just return the result of `_run_cmd_as_defer_to_thread()`.